### PR TITLE
Documentation and maintenance updates

### PR DESCRIPTION
**Pull request by kenman345**
Still working on this but figured it could go to a PR for now to give some insights as to whats happening.

### CODE REVIEW

**Comment by kenman345** [comment]
Keywords through command line isn’t finished yet. But otherwise this is pretty much feature complete

File: websites_schema.yml [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
             required: yes
             unique: yes
           "img":
             type: str
+            desc: Name for image file. Image should be 32x32px and end in .png
             pattern: /\.png$/i
           "bch":
             type: bool
             required: yes
+            desc: Do the accept BCH?
```
Comment (EchterAgo) [inline websites_schema.yml]: typo, should be they

File: websites_schema.yml [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
             pattern: /(true|false)/
           "btc":
             type: bool
+            desc: Do the accept BTC?
```
Comment (EchterAgo) [inline websites_schema.yml]: typo, should be they

File: websites_schema.yml [hunk 3]
```diff
@@ -1,33 +1,41 @@
...
             pattern: /(true|false)/
           "othercrypto":
             type: bool
+            desc: Do the accept Other Crypto?
```
Comment (EchterAgo) [inline websites_schema.yml]: typo, should be they

File: websites_schema.yml [hunk 4]
```diff
@@ -48,12 +56,14 @@ mapping:
             pattern: /\A([\w+\-].?)+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
           "status":
             type: str
+            desc: This should only be on a listing if its in the process of implementing BCH support but removed once it has completed. This is always a URL to the status of that effort.
```
Comment (EchterAgo) [inline websites_schema.yml]: typo, should be "it's"

**Comment by kenman345** [comment]
Okay so this PR needs some details:

First off I updated some documentation to inform people about the new fields and the new methods for adding data to the site.

These include:
- keywords tag
- adding an issue manually via command line
- adding a new category via command line

Along with these changes I deemed it necessary to create a `sections_schema.yml` file in the repository to base our section file off of. In this respect we have a means to validate the data and use against the new methods. Updating the schema will add in those additional validations or tags to the prompts in the `bundle exec rake add:category` method as well as ensure we are consistent. 

While doing this I did think about making a rake task and schema for the assets listings. These can remain for now unchecked as we don’t really expect more data to be adding with any frequency we should be concerned about. 

For now the method intended to add keywords to listings via command line is a bit of a shell. It will open a section file and display the listings by its identifier `name` and give numbers as well. The intent is to allow users to input which index to use for editing a listing and then being able to list off as many as you want, each new element that’s added will be separated with a prompt if they wish to add yet another keyword. I have not gotten to writing that part yet though as I also need to write that for the original manual entry rake task to use and wanted to template it out as it’s easier to figure out isolated with the keywords task than with the other task that takes a lot longer to dive into what I was trying to accomplish.

I may make one final change before merging this and that is to maybe move these tasks to a task file that rake would import. It’s a standard practice and Rake would automatically add in the file to the set of tasks just by the files very existence. Can always happen later, so we shall see if I get to that

**Comment by kenman345** [comment]
Forgot to mention:
- this Pr includes a change to delete out the generated `assets`folder from the site since we don’t presently use it and it’s 66kb css we don’t care about.
- this includes a robots.txt that on our beta site will prevent robot crawlers from hitting the beta site

File: Rakefile [hunk 5]
```diff
@@ -59,89 +59,77 @@ namespace :add do
...
+    tags['mapping'].each do |tag|
+      data = prompt_tag(tag[0], tag[1])
+      new_section[tag[0]] = data unless data.nil?
     end
-    a == 'y'
-  end
-  # rubocop:enable Semicolon
+    if valid_to_ins(categories, new_section, 'id')
+      categories[categories.count] = new_section
+      puts categories.count
```
Comment (zquestz) [inline Rakefile]: Should remove the puts.

Comment (zquestz) [reply Rakefile]: Or at least format it so it looks nice =)

File: verify.rb [hunk 6]
```diff
@@ -98,31 +81,73 @@ def process_section(section, validator)
...
+  errors = parser.errors()
+  return unless errors && !errors.empty?
+  errors.each do |e|
+    error(e.message.to_s)
+  end
+end
+
+def validate_alphabetical(set, identifier, set_name)
+  return unless set != (set_sort = set.sort_by { |s| s[identifier].downcase })
+  error("#{set_name} not ordered by #{identifier}. Correct order:" \
```
Comment (zquestz) [inline verify.rb]: Should use this style

```
str =<<-EOS
Whatever you want to print with #{interpolation} support
EOS
```

Comment (kenman345) [reply verify.rb]: You’re saying set a variable to that value first? Then send to error(..

Comment (zquestz) [reply verify.rb]: Yep.

**Comment by EchterAgo** [comment]
Is google75bd212ec246ba4f.html even still needed? Normally you can remove it after letting Google verifiy. I never had a reason to keep the file around.

**Comment by kenman345** [comment]
I hav no idea what its used for in first place, shall I just remove it?

**Comment by EchterAgo** [comment]
Verification from Google Webmaster Tools. To verify you own a site there you have to put a file like that into your website root. After it is verified the file is not needed anymore. So yea, I would remove it.

**Comment by EchterAgo** [comment]
Hrmmm, I just read the documentation and it says Google periodically checks it, so better not remove it. Weird, I have it removed on my sites, but Google Webmaster Tools still work.

https://support.google.com/webmasters/answer/35179?hl=en

**Pull request merged**